features/run/all

```diff
@@ -5,5 +5,5 @@ features/run/http-memory && \
 features/run/dom-http-memory && \
 features/run/database && \
 features/run/webdriver-memory && \
-features/run/webdriver-http-database && \
```

Comment:
If you look at the lego drawing, you'll see that the assemblies are currently named after the `_|⎻|_|⎻|_` legos. I quite like that as a naming convention - it summarises what legos are in the stack.

Reply:
Yeah I realised after doing this, we had already renamed to the opposite. What is confusing though, is that the `HttpTodoList` is not mentioned _in the assembly_ but it is, of course, used in the stack under test.

Reply:
Looking at it again, I'm confused. Why isn't `webdriver-memory` called `webdriver-dom-memory` then? and why isn't `webdriver-http-database` called `webdriver-dom-http-database` then? That would seem to be the naming convention you are suggesting?!

<img width="427" alt="screen shot 2017-11-07 at 17 09 02" src="https://user-images.githubusercontent.com/22017/32507079-65efec8c-c3de-11e7-8afa-b85326d439cc.png">

Reply:
Sticking with your names for now, so I can merge this. I'll open an issue to discuss further